docs/design/ovs-pipeline.md

```diff
@@ -53,62 +52,129 @@
+|               |             |                           | 0b10          | OutputToControllerRegMark       | Send packet to Antrea Agent.                                                             |
+|               | bits 25-32  | PacketInOperationField    | 0b00000001    | PacketInNPLoggingOperation      | Indicating packet needs logging for NetworkPolicy packetIn operation.                    |
+|               |             |                           | 0b00000010    | PacketInNPRejectOperation       | Indicating packet should be rejected for NetworkPolicy packetIn operation.               |
+|               |             |                           | 0b00000100    | PacketInNPStoreDenyOperation    | Indicating the corresponding connection has been dropped or rejected.                    |
+| NXM_NX_REG1   | bits 0-31   | TargetOFPortField         |               |                                 | Egress OVS port of packet.                                                               |
+| NXM_NX_REG2   | bits 0-31   | SwapField                 |               |                                 | Swap values in flow fields in OpenFlow actions.                                          |
+|               |             | PacketInTableField        |               |                                 | OVS table where the packet is decided to be sent to controller.                          |
+| NXM_NX_REG3   | bits 0-31   | EndpointIPField           |               |                                 | Field where stores selected Service Endpoint IPv4 address.                               |
+|               | bits 0-31   | APConjIDField             |               |                                 | Field to store Conjunction ID for Antrea Policy.                                         |
+| NXM_NX_REG4   | bits 0-15   | EndpointPortField         |               |                                 | Field stores IPv4 address of a Service's selected Endpoint.                              |
+|               | bits 16-18  | ServiceEPStateField       | 0b001         | EpToSelectRegMark               | Packet needs to do Service Endpoint selection.                                           |
+|               | bits 16-18  | ServiceEPStateField       | 0b010         | EpSelectedRegMark               | Packet has done Service Endpoint selection.                                              |
+|               | bits 16-18  | ServiceEPStateField       | 0b011         | EpToLearnRegMark                | Packet has done Service Endpoint selection and the selected Endpoint needs to be cached. |
+|               | bits 0-18   | EpUnionField              |               |                                 | The union value of EndpointIPField and EndpointPortField.                                |
```

Comment:
I think the description is invalid here: should it be `EpToSelectRegMark` instead of `EndpointIPField`?

Reply:
Good catch!